### DISCUSSION TITLE
Opening *.py.out.progress

### DISCUSSION THREAD

**Discussion by aliyyanurr** [phoebe-project/phoebe2#622] [Questions]
Hello, I am currently running MCMC of starspots parameter on external computing sources. I set the `progress_every_niters` to 100, so that I wish I could see how the MCMC runs even before it really ends. But I don't understand how too see the progress. How to open the *.py.out.progress in jupyter notebook? Is it possible to see the convergence of the progress using this *.py.out.progress file?

**Comment by gecheline** [comment]
Hi Aliyya,

If you've also set `detach=True` during the call to `b.run_solver()`, you can monitor the progress with `b.load_job_progress(solution='your_solution_tag')`. This will download the progress file from the server and load it into the bundle, so you can then plot and inspect the results from MCMC. 

If you haven't set `detach=True` and the process is freezing up your Jupiter notebook, it's more complicated because you would have to manually download the file from the server and load it in a separate notebook (probably in a new bundle, or a saved one from your prior work) with `b.import_solution()`.

If I'm running a long MCMC job, what I typically do is run the solver with `detach=True` and then save the bundle immediately after. So every time you want to check on the progress, you can load the saved bundle and use `b.load_job_progress()`.

Here are the links to the documentation for these two options:
[load_job_progress](http://phoebe-project.org/docs/2.4/api/phoebe.frontend.bundle.Bundle.load_job_progress)
[import_solution](http://phoebe-project.org/docs/2.4/api/phoebe.frontend.bundle.Bundle.import_solution)

**Reply by aliyyanurr** [reply]
Thank you Angela, I've tried the `import_solution` option (because I am using PHOEBE v2.3.54 and there is still no server option updates) and it works.
